proto/internal/temporal/server/api/archiver/v1/message.proto

```diff
@@ -63,6 +63,6 @@ message ArchiveVisibilityRequest {
     temporal.api.enums.v1.WorkflowExecutionStatus status = 9;
     int64 history_length = 10;
     temporal.api.common.v1.Memo memo = 11;
-    map<string, string> search_attributes = 12;
+    temporal.api.common.v1.SearchAttributes search_attributes = 12;
```

Comment:
This is breaking change but it is most likely safe (see another comment bellow).

Reply:
let us talk about this during the standup